invenio_communities/assets/semantic-ui/js/invenio_communities/community/new.js

```diff
@@ -31,17 +31,21 @@ import { CommunityApi } from "../api";
 import { communityErrorSerializer } from "../api/serializers";
 
 const IdentifierField = ({ formConfig, slug = "" }) => {
+  const [newSlug, setNewSlug] = React.useState(slug);
+
   const helpText = (
     <>
       {i18next.t(
         "This is your community's unique identifier. You will be able to access your community through the URL:"
       )}
       <br />
-      {`${formConfig.SITE_UI_URL}/communities/${slug}`}
+      {`${formConfig.SITE_UI_URL}/communities/${newSlug}`}
     </>
   );
+
   return (
     <TextField
+      onChange={e => { setNewSlug(e.target.value) }}
```

Comment:
Available in version `0.10.8`, but I made a change, I am passing only the value. Can you please change it to:
```suggestion
      onChange={value => { setNewSlug(value) }}
```